web/src/components/assets/AssetCreationForm.tsx

```diff
@@ -14,30 +14,13 @@ const AlertMessage = dynamic(() => import("uikit-react-public").then((mod) => mo
   ssr: false,
 });
 
-type AssetFormData = {
-  title: string;
-  description: string;
-  classification_impact: string;
-  protection: string;
-  legal_basis: string;
-  format: string;
-  expiry: string;
-  location: string[];
-  has_dspt: boolean;
-  stored_outside_uk_eea: boolean;
-  accessed_by_third_parties: boolean;
-  third_party_agreement: string;
-  status: string;
-};
-
 type AssetFormProps = {
-  onSubmit: (data: AssetFormData) => Promise<void>;
-  // setSelectedStudy: (study: Study | null) => void;
+  handleAssetSubmit: (data: AssetFormData) => Promise<void>;
   isSubmitting?: boolean;
 };
 
-export default function AssetForm(props: AssetFormProps) {
-  const { onSubmit, isSubmitting = false } = props;
+export default function AssetCreationForm(props: AssetFormProps) {
```

Comment:
I find this odd that it's not a modal like the study create form

Reply:
I think making them modals makes sense